### PR TITLE
Update impala-profile.md

### PR DESCRIPTION
**Pull request by tovganesh**
## Description & motivation
Update documentation to indicate way to turn on / off adapter instrumentation

### CODE REVIEW

File: website/docs/reference/warehouse-profiles/impala-profile.md [hunk 1]
```diff
@@ -116,6 +116,9 @@ Note: A typical setup of Cloudera EDH will involve the following steps to setup
 - Obtain keytab using kinit (kinit username@YOUR_REALM.YOUR_DOMAIN)
 - The keytab is valid for certain period after which you will need to run kinit again to renew validity of the keytab.
 
+### Instrumentation
+By default, the adapter will collect instrumentation events to help improve functionality and understand bugs. If you want to specifically switch this off, for instance, in a production environment, you can explicitly set the flag `usage_tracking: false` in your `profiles.yml` file. 
+
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/impala-profile.md]: what's the reason for adding this? Was a user concerned? Perhaps you can link to our existing docs section. But perhaps `usage_trackign` is an impala-specific setting?
```suggestion
### Instrumentation
By default, the adapter will send instrumentation events to Cloudera to help improve functionality and understand bugs. If you want to specifically switch this off, for instance, in a production environment, you can explicitly set the flag `usage_tracking: false` in your `profiles.yml` file.

Relatedly, if you'd like to turn off dbt Lab's anonymous usage tracking, see [YAML Configurations: Send anonymous usage stats](https://docs.getdbt.com/reference/global-configs#send-anonymous-usage-stats) for more info
```

**Review by dataders — CHANGES_REQUESTED** [review]
just one comment!

File: website/docs/reference/warehouse-profiles/impala-profile.md (hunk 1 above)
Comment (tovganesh) [reply website/docs/reference/warehouse-profiles/impala-profile.md]: Yup, this is impala specific setting.

**Review by dataders — APPROVED** [review]
thanks!

**Pull request merged**